src/NuGet.Core/NuGet.Commands/RestoreCommand/RequestFactory/DependencyGraphSpecRequestProvider.cs

```diff
@@ -156,6 +156,7 @@ private static ExternalProjectReference GetExternalProject(PackageSpec rootProje
             var globalPath = GetPackagesPath(restoreArgs, projectPackageSpec);
             var settings = Settings.LoadImmutableSettingsGivenConfigPaths(projectPackageSpec.RestoreMetadata.ConfigFilePaths, settingsLoadingContext);
             var sources = restoreArgs.GetEffectiveSources(settings, projectPackageSpec.RestoreMetadata.Sources);
+            var namespaces = PackageNamespacesConfiguration.GetPackageNamespacesConfiguration(settings);
```

Comment:
Are you to use it in future tasks? Currently it's assigned value, but not using it.

Reply:
I reverted this change from this PR. I will add this line of code in my follow-up for package namespaces.